src/api/schema/mod.rs

```diff
@@ -5,10 +5,10 @@ mod health;
 mod meta;
 mod metrics;
 mod relay;
-mod scalar;
 pub mod sort;
 
 use async_graphql::{EmptyMutation, MergedObject, MergedSubscription, Schema, SchemaBuilder};
+pub use vector_core::api::schema::scalar;
```

Comment:
It doesn't appear `scalar` was `pub` before. Is the re-export necessary now?

Reply:
I believe so, but I will follow-up.